.changeset/young-news-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopify/shopify-api": minor
+---
+
+Add helpers to convert unified admin URLs, and sanitizeShop util method can now support unified admin URLs
```

Comment:
This will be in the changelog, so just checking: is "unified admin" a term that will be clear to everyone? Not sure if we mention it in our docs.

Reply:
How about half-way to `ShopAdminUrl` ?

Reply:
Love it.